features/e2e_tests/e2e_tests_for_visa_checkout.feature

```diff
@@ -24,12 +20,80 @@ Feature: Visa checkout E2E tests
       | VISA_FRICTIONLESS     |
       | VISA_NON_FRICTIONLESS |
 
-  @base_config
   Scenario: Declined Authentication by Visa checkout using declined visa card
-    Given User clicks on Visa Checkout button
+    Given JS library is configured with VISA_CHECKOUT_CONFIG and BASE_JWT
+    And User opens example page
+    And User clicks on Visa Checkout button
+    And User fills visa checkout email address
+    And User fills visa checkout one time password
+    When User select VISA_DECLINED_CARD card on visa checkout popup
+    And User confirm displayed card with data
+    Then User will see payment status information: "Decline"
+    And User will see that notification frame has "red" color
+
+  Scenario: Successful Authentication by Visa checkout with submit on success config
+    Given JS library is configured with VISA_CHECKOUT_WITH_SUBMIT_ON_SUCCESS_CONFIG and BASE_JWT
+    And User opens example page
+    And User clicks on Visa Checkout button
+    And User fills visa checkout email address
+    And User fills visa checkout one time password
+    When User select VISA_CARD card on visa checkout popup
+    And User confirm displayed card with data
+    Then User will see payment status information: "Payment has been successfully processed"
```

Comment:
For this 'Scenario: Successful Authentication by Visa checkout with submit on success config' expected result should be And User will be sent to page with url "www.example.com" having params.
We will not see notification frame after redirection